website/client/src/components/groups/invitationListModal.vue

```diff
@@ -0,0 +1,271 @@
+<template>
+  <b-modal
+    id="invitation-list"
+    size="md"
+    :hide-header="true"
+    :hide-footer="true"
+  >
+    <div class="dialog-close">
+      <close-icon @click="close()" />
+    </div>
+    <h2 class="text-center textCondensed" v-once>
+      {{ $t('invitations') }}
+    </h2>
+    <div
+      v-for="member in members"
+      :key="member._id"
+      class="member-row"
+    >
+      <div class="class-icon">
+        <class-badge
+          v-if="member.stats"
+          :member-class="member.stats.class"
+          style="--badge-size: 40px"
+        />
+      </div>
+      <div class="usernames">
+        <user-label :user="member" class="user-label" /> <br>
+        <span class="username">
+          @{{ member.auth.local.username }}
+        </span>
+      </div>
+      <div :class="{
+        'status': true,
+        'accepted': member.accepted === true,
+        'declined': member.accepted === false,
+        'pending': member.accepted === null
+      }">
+        <div
+          v-if="member.accepted === true"
+          class="accepted float-right"
+        >
+          {{ $t('accepted') }}
+        </div>
+        <div
+          v-if="member.accepted === false"
+          class="declined float-right"
+        >
+          {{ $t('declined') }}
+        </div>
+        <div
+          v-if="member.accepted === null"
+          class="pending float-right"
+        >
+          {{ $t('pending') }}
+        </div>
+        <div class="circle">
+          <div
+            v-if="member.accepted === true"
+            class="svg-icon color"
+            v-html="icons.check"
+          ></div>
+          <div
+            v-if="member.accepted === false"
+            class="svg-icon color"
+            v-html="icons.close"
+          ></div>
+        </div>
+      </div>
+    </div>
+  </b-modal>
+</template>
+
+<style lang='scss'>
+  #invitation-list {
+    .modal-header {
+      background-color: #edecee;
```

Comment:
Would it be possible to get these into the `scoped` section so we can use the `colors.scss` variables instead of hardcoded hex values?

Reply:
we can still use the colors here - It didnt worked with the scoped version even with ::v-deep - here always get into the conditional bootrap restyling for stuff  - which not always works in a scoped way